_includes/cwl/metadata_example2.cwl

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env cwl-runner
+
+class: CommandLineTool
+id: Example tool
+label: Example tool
+cwlVersion: v1.0
+doc: |
+    An example tool demonstrating metadata. Note that this is an example and the metadata is not necessarily consistent.
+
+requirements:
+  - class: ShellCommandRequirement
+
+inputs:
+  bam_input:
+    type: File
+    doc: The BAM file used as input
+    format: edam:format_2572
+    inputBinding:
+      position: 1
+
+stdout: output.txt
+
+outputs:
+  report:
+    type: File
+    format: edam:format_1964
+    outputBinding:
+      glob: "*.txt"
+    doc: A text file that contains a line count
+
+baseCommand: ["wc", "-l"]
+
+$namespaces:
+  s: https://schema.org/
+  edam: http://edamontology.org/
+
+$schemas:
+- https://schema.org/docs/schema_org_rdfa.html
+
+s:author:
+  - class: s:Person
+    s:id: https://orcid.org/0000-0002-6130-1021
+    s:email: dyuen@oicr.on.ca
```

Comment:
yes. actually, I prefer that anyway because including the protocol is always helpful. Will make the relevant changes now